src/DynamoCoreWpf/Views/About/AboutWindow.xaml.cs

```diff
@@ -37,6 +51,130 @@ private void HandleEsc(object sender, KeyEventArgs e)
                 Close();
         }
 
+        private void LoadHTMLAboutBoxContent()
+        {
+            string executingAssemblyPathName = Assembly.GetExecutingAssembly().Location;
+            string rootModuleDirectory = Path.GetDirectoryName(executingAssemblyPathName);
+            var htmlLicensePath = Path.Combine(rootModuleDirectory, "third_party_licenses_html");
```

Comment:
So these htmls will be delivered instead of being dll resources, did I understand it correct?

Reply:
Thank you for your answer @mjkkirschner I think this goes with my earlier concern, I was hoping they can be delivered with the npm package and embedded into project dll resources, just like the js file so we dont have to introduce new files. But I understand you started this with the way aligning more with the licence.rtf. I am fine with this and seems we can remove file at a later Global launch case.

Reply:
hmm, I see, I can try packing these files into an assembly or assemblies and extracting from there... maybe we can use a simple naming scheme to automatically locate licenses that are embedded in any assembly, that way we don't need to have dependencies in dynamoCore on extension folders or files...

will think about it.

Reply:
You can start from Notification Center, it will be more straightforward. If we update the cp step there, the html should be included automatically by Github Publish Action. Then it is only about how to consume it on Dynamo end. My 2 cents. I am supportive of using same naming or some schema, so DynamoCore only need to traverse all dlls and too out these block and append them one by one